### ISSUE TITLE
Proposed changes to top level localization terms

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#15362]
There are some inconsistencies at the top level in the localization branch. 

**CURRENT DEFINITIONS:** 
 (1) GO:0051179 localization 
 Any process in which a cell, a substance, or a cellular entity, such as a protein complex or organelle, is transported, tethered to or otherwise maintained in a specific location. In the case of substances, localization may also be achieved via selective degradation.

-- (2) GO:0051234 establishment of localization (is a localization)
Any process that localizes a substance or cellular component. This may occur via movement, tethering or selective degradation. 

---- (3) GO:0006810 	transport (is a 'establishment of localization')
The directed movement of substances (such as macromolecules, small molecules, ions) or cellular components (such as complexes and organelles) into, out of or within a cell, or between cells, or within a multicellular organism by means of some agent such as a transporter, pore or motor protein. 

-- (4) GO:0051235 maintenance of location (is a localization)
Any process in which a cell, substance or cellular entity, such as a protein complex or organelle, is maintained in a location and prevented from moving elsewhere. 

---
'Establishment of localization' should not have 'tethering or selective degradation' in the definition, this should belong to 'maintenance'. 

Proposed changes: 
1. GO:0051234 establishment of localization : Remove second sentence in definition, to keep just "Any process that localizes a substance or cellular component _to a specific location_". 

2. GO:0051235 maintenance of location: add the the tethering or degradation part of the 'establishment of localization' def, to create this new definition: 
"Any process in which a cell, substance or cellular entity, such as a protein complex or organelle, is maintained in a location and prevented from moving elsewhere. This may occur via tethering or selective degradation." (are there other possible mechanisms by which a protein can maintain its localization? ) 

If that's OK I'll implement this, and fix #15360 accordingly. 

Thanks, Pascale

**Comment by ValWood** [comment]
I think "establishment" can also include "tethering"....

see  my comment in  #15360

**Comment by pgaudet** [comment]
Hi @ValWood (any anyone else!) 

Are you proposing to merge establishment of location with maintenance of location? 
One possibility is to merge the top level terms and keep children labels as they are (and/or do further merges as required). 

What do you think? 

Pascale

**Comment by ValWood** [comment]
> Are you proposing to merge establishment of location with maintenance of location?

No, it's just that I'm not sure they have the same meaning for every process, unfortunately....
(sometimes establishment of localization is transport, sometimes it is tethering). Or that they are both required for every process. Sometimes maintenance is  a process, sometimes it might not be....I'm not sure.

Possibly if we use the terms biologists use and define accordingly
(for example, "spindle pole body insertion into the nuclear envelope" instead of  "establishment of spindle pole body location to the nuclear envelope", it becomes easier to tease apart the process we mean. 

But I'm not sure.....
Sorry not to be more helpful....

**Comment by pgaudet** [comment]
Discussion with Val: 

- We are not aware of any example of **selective degradation** leading to localization. Does anyone have an example? Otherwise it might be less confusing to remove this part of the definition. 
The only case we could think of was if a protein that sequesters a target gets degraded, then that target is free to move and localize somewhere - was this the intended usage ? If it's the case that would be better captured by 'regulation of localization'.
